wavefront/resource_ingestion_policy.go

```diff
@@ -59,7 +59,7 @@ func resourceIngestionPolicyRead(
 	d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*wavefrontClient).client.IngestionPolicies()
 	id := d.Id()
```

Comment:
I don't think you need the id variable since you don't need a string pointer anymore.